src/Compilers/VisualBasic/Portable/Binding/Binder_Utils.vb

```diff
@@ -1132,7 +1149,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End If
 
             Select Case container.TypeKind
-                Case TypeKind.Module
+                Case TYPEKIND.Module
```

Comment:
I didn't understand why the capitalization changed. `TypeKind` seemed correct.

Reply:
I don't think I did this.

---
In reply to: [137607034](https://github.com/dotnet/roslyn/pull/21742#discussion_r137607034) [](ancestors = 137607034)

Reply:
My guess is the VB auto-formatting did this, but I don't understand why it would.